packages/eui/src/components/steps/step.tsx

```diff
@@ -81,12 +81,14 @@ export const EuiStep: FunctionComponent<EuiStepProps> = ({
   ];
   const cssTitleWrapperStyles = titleStyles.euiStep__titleWrapper;
 
+  const size = titleSize === 'xxs' ? 'xs' : titleSize;
+
   return (
     <div className={classes} css={cssStyles} {...rest}>
       <div className="euiStep__titleWrapper" css={cssTitleWrapperStyles}>
         <EuiStepNumber number={step} status={status} titleSize={titleSize} />
         <EuiTitle
-          size={titleSize as EuiTitleSize}
+          size={size}
```

Comment:
Do we need the size const or can we just inline the ternary?
```suggestion
          size={titleSize === 'xxs' ? 'xs' : titleSize}
```

Reply:
Updated [here](https://github.com/elastic/eui/pull/7813/commits/0de2968c7eb011083111aee2e5671cf821f57e47#diff-1979bbaea69e216a29ac928a414ef72a6f532f5bbf16078fff03e5ef7ce9bf02R95)